coordinator/src/main/scala/filodb.coordinator/queryplanner/LogicalPlanUtils.scala

```diff
@@ -235,23 +234,10 @@ object LogicalPlanUtils extends StrictLogging {
     // SubqueryWithWindowing has such a lookback while TopLevelSubquery does not.
     logicalPlan match {
       case sww: SubqueryWithWindowing => getLookBackMillis(sww.innerPeriodicSeries).map(lb => lb + sww.subqueryWindowMs)
-      case _ => {
-        val staleDataLookbackMillis = WindowConstants.staleDataLookbackMillis
-        val leaf = LogicalPlan.findLeafLogicalPlans(logicalPlan)
-        val valToReturn = {
-          if (leaf.isEmpty) Seq(0L) else {
-            leaf.map { l =>
-              l match {
-                case lp: RawSeries => lp.lookbackMs.getOrElse(staleDataLookbackMillis)
-                case _             => 0
-              }
-            }
-          }
-        }
-        valToReturn
-      }
```

Comment:
Old setup did not correctly handle cases like:
```
rate(foo{}[10m:30s])
```
because the `SubqueryWithWindowing` was not at the root. We need to know if SQWW exists anywhere in the tree, because its lookback/offset aren't propagated to its children.